packages/fxa-content-server/server/lib/routes/get-openid-configuration.js

```diff
@@ -23,6 +24,7 @@ const openidConfig = {
   revocation_endpoint: revocationEndpoint,
   token_endpoint: tokenEndpoint,
   userinfo_endpoint: userInfoEndpoint,
+  verify_endpoint: verifyEndpoint,
```

Comment:
Unfortunately `verify_endpoint` isn't defined in OAuth 2 or OpenID Connect.

Reply:
Ah, yes thanks for pointing this out. We discussed it in sprint planning and it seemed straight forward, but you are right, this field isn’t actually specified. 

I'll follow up with the filer of this issue and try to get a better handle on why this is being requested. I just looked up the specs for [openid connect](https://openid.net/specs/openid-connect-registration-1_0.html) and [oauth](https://datatracker.ietf.org/doc/html/rfc8414#section-3.2), both of which indicate additional claims/parameters MAY be included in this response. That being said, adding non-standard properties is probably not desirable.

Reply:
Good call, @chenba .  I didn't think of that.

Reply:
@clouserw not really sure what to do here. Should we reject? I talked to the issue filer and they basically have to do something kind of hacky in order to determine the verify endpoint URL. So I see why they would like this endpoint to be specified.

@chenba concerning my feedback, are you sure it's invalid to add additional properties to this response? I certainly don't want to break anything, but it doesn't appear to be invalid to include an additional property. If I am misinterpreting that spec, or there's something else that indicates you shouldn't do this, please let me know.

Reply:
Unless you feel strongly about rejecting this entirely, I am of the opinion that we land this. I think it's logical that someone would look to our docs in the event they do not see this claim in the specification. It also seems that this is a useful end point so shining a little light on it probably doesn't hurt. We also noticed that other specs will include values that aren't specified. For example, check [this one](https://login.microsoftonline.com/common/.well-known/openid-configuration) out. The `kerberos_endpoint` isn't part of the openid connect spec.

Reply:
Sorry, didn't realize my approval's needed here.  Definitely couldn't hurt to add it.  👍